java/src/main/java/com/logicalclocks/hsfs/FeatureView.java

```diff
@@ -130,6 +130,11 @@ public void delete() throws FeatureStoreException, IOException {
     featureViewEngine.delete(this.featureStore, this.name, this.version);
   }
 
+  public static void clean(FeatureStore featureStore, String featureViewName, Integer featureViewVersion)
```

Comment:
I wonder, wouldn't is be better to add a function like `deleteFeatureView(String featureViewName, Integer featureViewVersion)` to the `FeatureStore` class?

Reply:
Yea, I considered this option as well. But the convention of the API is that we keep delete in an entity e.g fg, td, fv. And `feature_store` just has `get_*` methods.

Reply:
Okay, makes sense